src/style.css

```
@@ -1,3 +1,77 @@
+html {
+  font-family: Roboto, sans-serif;
+  --main-color: #00acdc;
+}
+
 body {
+  margin: 0 auto;
+}
+
+.container {
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+  padding-left: 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  height: 60px;
+  flex-shrink: 0;
+}
+
+.img {
```

```suggestion
.logo__img {
```
The class should have a meaningful name